frontend/pulse3d/src/components/interactiveAnalysis/InteractiveAnalysisModal.js

```diff
@@ -562,33 +580,80 @@ export default function InteractiveWaveformModal({
         originalData,
         changelog,
         peakValleyWindows,
+        peakY1,
+        peakY2,
+        valleyY1,
+        valleyY2,
       })
     );
   };
 
   const updateChangelog = () => {
     let changelogMessage;
-
     // changelog will have length of 0 if a user Undo's until initial state
     if (changelog[selectedWell] && changelog[selectedWell].length > 0 && markers.length === 2) {
+      //If Change log has changes
       const wellChanges = changelog[selectedWell];
-      const { peaks, valleys, startTime, endTime, pvWindow } = wellChanges[wellChanges.length - 1];
-
-      changelogMessage = getChangelogMessage(peaks, valleys, startTime, endTime, pvWindow);
+      //Get snapshot of previus state
+      const {
+        peaks,
+        valleys,
+        startTime,
+        endTime,
+        pvWindow,
+        valleyYOne,
+        valleyYTwo,
+        peakYOne,
+        peakYTwo,
+      } = wellChanges[wellChanges.length - 1];
+      changelogMessage = getChangelogMessage(
```

Comment:
With this many parameters to a function I think it makes sense to just pass in an object instead of each indivodual value